server/methods.coffee

```diff
@@ -178,9 +178,13 @@ Meteor.methods
     incidents = createIncidentReportsFromEnhancements(enhancements, options)
     incidents = incidents.map (incident) ->
       incident = _.pick(incident, incidentReportSchema.objectKeys())
+    # check for unexpected urls
+    if not options.url.startsWith("promedmail.org/post/")
```

Comment:
That wouldn't match anything because the $ requires the match to go to the end of the field.